src/commcare_cloud/ansible/roles/citusdb/meta/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+dependencies:
+  - role: monit
+  - role: postgresql_base
+    postgresql_version: '11'
+    postgresql_process_path: /var/run/postgresql
+    postgresql_cluster_name: '11/main'
```

Comment:
why do these need to be overridden?

Reply:
default cluster name in `11/main` if it is not mentioned the `postgresql` fails to restart. 
in `bionic` the process path in `system service` file is `/var/run/postgresql`, again it was failing to start when i changed it to `/opt/data/postgresql`. It would be a good idea to set this on distribution type instead of passing as variable .

Reply:
OK, nice find. I had this issue when testing on bionic but never got to the bottom of it.